AsyncDisplayKit/ASDisplayNode.mm

```diff
@@ -1327,7 +1378,7 @@ - (void)measureNodeWithBoundsIfNecessary:(CGRect)bounds
   {
     ASDN::MutexLocker l(__instanceLock__);
     supportsRangedManagedInterfaceState = [self supportsRangeManagedInterfaceState];
-    hasDirtyLayout = [self _hasDirtyLayout];
+    hasDirtyLayout = _calculatedDisplayNodeLayout->isDirty();
```

Comment:
I think we should nil check here
`hasDirtyLayout = _calculatedDisplayNodeLayout == nil ? NO : _calculatedDisplayNodeLayout.isDirty();`